frontend/src/components/Board.vue

```diff
@@ -157,8 +178,26 @@ export default class Board extends Vue {
     this.vertexManager.draw(vertex);
   }
 
+  startPencil(event: KonvaMouseEvent) {
```

Comment:
Since pencil drawing will be probably used only on pencil drawing mode, I think u can try to move this methods `startPencil` and `movePencil` to `pencil_manager`.